central/notifier/service/service_impl.go

```diff
@@ -186,7 +200,14 @@ func (s *serviceImpl) TestUpdatedNotifier(ctx context.Context, request *v1.Updat
 	if err := s.reconcileUpdateNotifierRequest(ctx, request); err != nil {
 		return nil, err
 	}
-	notifier, err := notifiers.CreateNotifier(request.GetNotifier())
+	if request.GetUpdatePassword() {
+		err := notifierUtils.SecureNotifier(request.GetNotifier(), s.cryptoKey)
+		if err != nil {
+			// Don't send out error from crypto lib
+			return nil, errors.New("Error securing notifier")
```

Comment:
nit: error encrypting notifier credentials (can make this change with unit tests PR)

Reply:
I will make this change in a followup unit test PR.